src/datasources/tag/components/TagQueryEditor.test.tsx

```diff
@@ -6,6 +6,13 @@ import { TagDataSource } from '../TagDataSource';
 import { TagQuery, TagQueryType } from '../types';
 import { TagQueryEditor } from './TagQueryEditor';
 
+jest.mock('@grafana/runtime', () => ({
+  ...jest.requireActual('@grafana/runtime'),
+  getTemplateSrv: () => ({ getVariables: getVariablesMock }),
+}));
+
+const getVariablesMock = jest.fn(() => [{ name: 'testVar' }]);
```

Comment:
If you implement the approach in my other comment, this won't work anymore. But it should actually be simpler. We can add the mock implementation for `getVariables` here:

https://github.com/ni/systemlink-grafana-plugins/blob/335fb5f56938ce33c34993df5056440e1a58a245/src/test/fixtures.ts#L21-L23